Python/Product/PythonTools/PythonTools/LanguageServerClient/FileWatcher/Listener.cs

```diff
@@ -92,6 +92,9 @@ public void AddPatterns(FileSystemWatcher[] patterns) {
             Array.ForEach(patterns, p => AddPattern(p));
         }
 
+        public void AddExclude(FileSystemWatcher pattern) {
+            _matcher.AddExclude(pattern.GlobPattern);
+        }
```

Comment:
current downside with this change is that exclude list isn't trimmed or cleared and rebuilt until workspace changes.

so for every interpreter change we just add more excludes